### PR TITLE
DDCNLS-3849 bumped version numbers, added paramater for s3

### PR DESCRIPTION
**Pull request by john-best1**

### CODE REVIEW

File: app/views/templates/error_template.scala.html [hunk 1]
```diff
@@ -29,7 +29,7 @@ <h1>@heading</h1>
 
 
 @{
-    templateRenderer.renderDefaultTemplate(layouts.article(mainContent), Map[String, Any](
+    templateRenderer.renderDefaultTemplate(config.ApplicationConfig.frontendTemplatePath, layouts.article(mainContent), Map[String, Any](
```
Comment (daniel-manning) [inline app/views/templates/error_template.scala.html]: appConfig is being used for some reason, may as well use it here

File: app/views/templates/tamc_main.scala.html [hunk 2]
```diff
@@ -171,7 +171,7 @@
 
 }
 @{
-    templateRenderer.renderDefaultTemplate(layouts.article(mainContent, false, None), Map[String, Any](
+    templateRenderer.renderDefaultTemplate(ApplicationConfig.frontendTemplatePath, layouts.article(mainContent, false, None), Map[String, Any](
```
Comment (daniel-manning) [inline app/views/templates/tamc_main.scala.html]: same here

Comment (john-best1) [reply app/views/templates/tamc_main.scala.html]: done

File: app/views/templates/error_template.scala.html (hunk 1 above)
Comment (john-best1) [reply app/views/templates/error_template.scala.html]: done

**Pull request merged**